src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx

```diff
@@ -121,7 +121,7 @@ export interface CurrencyInputPanelProps extends WithClassName {
   onUserInput: (value: string) => void
   onMax?: () => void
   showMaxButton: boolean
-  label?: string
+  label?: ReactNode
```

Comment:
Mod: change `label` type from `string` to `ReactNode` to pass components

Reply:
These comments are useful!
But they will get lost in the PR.
Can you add them to the code?

Reply:
I think anxo pointed out adding the mod comments to the code makes it hard to compare line by line so we decided to stop